scripts/ci/update-github

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -ex
+
+TAG_NAME=$(git tag -l --contains HEAD)
+
+rm -rf dcos-nightly
+
+# Setup Git
+git config --global user.email "$GIT_USER@users.noreply.github.com"
+git config --global user.name "MesosphereCI Robot"
+git config credential.helper 'cache --timeout=300'
+git clone "https://$GIT_USER:$GIT_PASSWORD@github.com/mesosphere/dcos" dcos-nightly
+cd dcos-nightly
+
+if [[ $TAG_NAME ]]
+then
+  git checkout -- .
+  git remote add upstream "https://$GIT_USER:$GIT_PASSWORD@github.com/dcos/dcos"
+  git checkout master
+  git checkout -b dcos-ui/$TAG_NAME
+
+  cp -fR ../buildinfo.json packages/dcos-ui/buildinfo.json
+
+  git add "packages/dcos-ui/buildinfo.json"
+  git commit -m "chore(dcos-ui): update package"
+  git push --force origin dcos-ui/$TAG_NAME
+
+  cd ../
+
+  CHANGELOG=$(TAG_NAME=$TAG_NAME ./scripts/ci/generate-changelog)
+
+  cd dcos-nightly
+
+  TEMPLATE=$(curl -H "Accept: application/vnd.github.v3.raw" https://api.github.com/repos/dcos/dcos/contents/PULL_REQUEST_TEMPLATE.md)
+  TASKS_HEAD="Corresponding DC/OS tickets (obligatory)"
+
+  PR_BODY=$(echo "${TEMPLATE}" | sed "s@${TASKS_HEAD}@${TASKS_HEAD}<br /><br />${CHANGELOG}@")
+  PR_BODY_NO_NEWLINES=$(echo "${PR_BODY//$'\n'/<br />}")
+  PR_BODY_NO_NEWLINES_NO_HEADING=$(echo "${PR_BODY_NO_NEWLINES//$'#'/}")
+
+  pullRequest="{
+    \"title\": \"chore(dcos-ui): update package\",
+    \"body\": \"${PR_BODY_NO_NEWLINES_NO_HEADING}\",
+    \"head\": \"mesosphere:dcos-ui/$TAG_NAME\",
+    \"base\": \"master\"
+  }"
+
+  curl -i -u "$GIT_USER:$GIT_PASSWORD" https://api.github.com
+  curl -H "Content-Type: application/json" -u "$GIT_USER:$GIT_PASSWORD" -X POST -d "${pullRequest}" https://api.github.com/repos/dcos/dcos/pulls
+else
+  git remote add upstream "https://$GIT_USER:$GIT_PASSWORD@github.com/dcos/dcos"
+  git checkout -- .
+  git checkout dcos-ui/latest
+  git pull --rebase -X theirs upstream master
+
+  cp -fr ../buildinfo.json packages/dcos-ui/buildinfo.json
+
+  git add "packages/dcos-ui/buildinfo.json"
+  git commit --amend --no-edit
+  git push --force origin dcos-ui/latest
```

Comment:
I think we should use `--force-with-lease` here too

Reply:
Switched to use it.